modules/02_welfare/utilitarian/postsolve.gms

```diff
@@ -17,8 +17,21 @@ p02_inconvPen_Switch_Track(t,regi) = (sum((entySe,entyFe,te,sector,emiMkt)$(se2f
 																		/ vm_cons.l(t,regi);	
 $ENDIF.INCONV_bioSwitch
 
+*** track inconvenience penalty for no-energy bio/synfuel switching to check
+*** how large it is relative to consumption
+p02_inconvPen_NonEn_Switch_Track(t,regi)
+  = ( sum((entySe,entyFe,te,sector,emiMkt)$(
+                            se2fe(entySe,entyFe,te) 
+                        AND entyFe2sector2emiMkt_NonEn(entyFe,sector,emiMkt)
+                        AND (entySeBio(entySe) OR entySeSyn(entySe) )        ),
+        v02_NegInconvPenNonEnSwitch.l(t,regi,entySe,entyFe,sector,emiMkt)
+      + v02_NegInconvPenNonEnSwitch.l(t,regi,entySe,entyFe,sector,emiMkt)
```

Comment:
As above: why is this two times the negative variable, not positive and negative?

Reply:
I'm running tests now to investigate the effect of fixing this mistake